cdragontoolbox/__main__.py

```diff
@@ -74,6 +74,8 @@ def parse_storage_args(parser, args) -> Storage:
             conf['cdn'] = cdn
         elif cdn != 'default':
             parser.error("--cdn is only supported for 'rads' storage")
+        if conf['type'] == 'patcher':
+            conf['patchline'] = args.patchline
```

Comment:
An error should be thrown if `--patchline` is set and storage type is not `patcher` (as for `--cdn`).
This is easy to do if `--patchline` defaults to `None` as suggested.